avalon/nuke/pipeline.py

```diff
@@ -1,43 +1,130 @@
-"""Pipeline functionality specifically for Nuke
+import os
+import importlib
+from .. import api, io
+import contextlib
+from pyblish import api as pyblish
+from ..vendor import toml
+import nuke
 
-Hi Arvid!
+from ..vendor.cgLogging import getLogger as nLogger
```

Comment:
Could you drop a few lines about why you wanted to include `cgLogging` with this PR? What problem(s) were you having that is solved with this library?

Also, let's stick to `snake_case` for our internal members, e.g. `n_logger` or `cg_logger` etc.